.github/workflows/playwright.yml

```
@@ -33,12 +33,14 @@ jobs:
           cache: "pnpm"
       - name: Install dependencies
         run: pnpm install
+      - name: Build production dist/ for previewing
+        run: pnpm build
       - name: Install Playwright Browsers
         run: pnpm exec playwright install --with-deps
       - name: Run Playwright tests
         run: pnpm exec playwright test
       - uses: actions/upload-artifact@v3
-        # if: github.ref == 'refs/heads/main'
+        if: always()
```

```suggestion
        if: ${{ always() }}
```

ref: https://docs.github.com/en/actions/learn-github-actions/expressions#always